frontend/src/sass/elements/_accordions.scss

```diff
@@ -0,0 +1,21 @@
+.fs-accordion {
```

Comment:
Also since the rules are no longer in accordion, do we want to remove?

Reply:
Removed